### PR TITLE
Enable "enumerate" in sssd.conf on slurm nodes.

### PR DESCRIPTION
**Pull request by enasca**
Requested by @ryade76. I've created a new role and a new host group that we can use to add other slurm-related tasks in the future.

### CODE REVIEW

File: ansible/roles/slurm_node/templates/01_slurm.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[domain/ikim.uk-essen.de]
```
Comment (hfrSchmidt) [inline ansible/roles/slurm_node/templates/01_slurm.conf.j2:1]: Just curious: Do we not have a variable containing our domain name?
Having this hard-coded in the role doesn't seem good.

Comment (hfrSchmidt) [inline ansible/roles/slurm_node/templates/01_slurm.conf.j2:1]:
```suggestion
[domain/{{ groups['slurm_nodes'][0].split('.', 1)[1] }}]
```
Could work as well, assuming that the entry in the inventory is in the form [host].[domain]

File: ansible/roles/slurm_node/tasks/sssd_enumerate.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+- name: Enable the enumerate option in sssd.conf
+  ansible.builtin.template:
+    src: 01_slurm.conf.j2
+    dest: /etc/sssd/conf.d/01_slurm.conf
+    owner: root
+    group: root
+    mode: 0644
```
Comment (hfrSchmidt) [inline ansible/roles/slurm_node/tasks/sssd_enumerate.yml:7]: It would be best to also add a notify + handler to restart the sssd service on config change.

**Review by hfrSchmidt — CHANGES_REQUESTED** [review]

File: ansible/roles/slurm_node/tasks/sssd_enumerate.yml (hunk 2 above)
Comment (enasca) [reply ansible/roles/slurm_node/tasks/sssd_enumerate.yml:7]: Added now.

**Review by hfrSchmidt — APPROVED** [review]

**Pull request merged**